src/homepage/_posts/2017-06-20-james-3.0.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "Apache James 3.0 release"
+date:   2017-06-20 11:30:00 +0700
+categories: james update
+---
+
+**Apache James version 3.0** have been released.
+
+After several years of efforts from the community, the Apache James developers are delighted to announce you the release
+of the **3.0 version** of the Apache James server.
+
+This release brings:
+ - **New protocols**: IMAP, [JMAP](https://jmap.io/) (experimental), .
```

Comment:
we are not sure if only JMAP is experimental or both are.
I would separate IMAP and JMAP by a new line